site_tmpl/projectionlist_detail.html

```diff
@@ -46,7 +46,7 @@ <h3> Projectionists In Training </h3>
     {% endfor %}
     
 </table>
-<h3> Licensed Projectionists </h3>
+<h3> Passed Practical Projectionists </h3>
```

Comment:
I would consider leaving this as is (since "Passed Practical" and "Licensed" are not actually the same thing). Instead I would recommend wrapping this in an if statement that checks to see if anyone is licensed and only displays this section if that's the case. That would give us the most flexibility in the event that we decide to start licensing our members again.